### ISSUE TITLE
[Sprint 4 7/11/20] Scrum Updates

### ISSUE THREAD

**Issue by anhyesun** [CPSC436/ThinkSmall#50]
- What did you work on this past iteration (2 weeks)?
- What were any major issues/challenges you ran into?
- What do you plan to work on for this coming iteration (2 weeks)?

**Comment by yehee** [comment]
@alice-0-kim Alice Kim (d7q0b)

**What I did**
Worked on DB/Server set up + implementing API endpoints (#55), API documentation
Requested for PR reviews and provided feedback for suggestions
Revisited minimal/standard requirements and worked on re-scoping
Attended team meeting

**Issues/challenges**
Trying to correctly prioritize tasks
Implementing best practices for keeping clean code
Avoiding unnecessary merge conflicts

**What I will work on**
Start working on deployment and making sure that the authentication is working on production mode
Continue working on editing/deleting `users`/`businesses`/`requests`
Create additional issues as the project progresses

**Comment by johnskimca** [comment]
@johnskimca - John Kim - q6i8

What I did
Created login and signup components, connecting them to redux actions (pr still required for the redux actions which adds user to database) #41 
Created a slack post detailing project setup (.env.locals, where to yarn install)
Connected map to redux #70 

Issues/challenges
Avoiding unnecessary merge conflicts
Trying to be more active on slack

What I will work on
Creating actions that allow getting user by either email or id #79 
Assignment 5
Implementing local-passport authentication

**Comment by anhyesun** [comment]
@anhyesun - Hyesun An - c5g2b 
**What I did** 
Created issues for this sprint and organized team meetings, did code reviews for most of PRs
Researched a way of user authentication like OAuth and image storing file system such as Amazon S3.
Implemented google OAuth2.0 with implicit grant and used passport for server-side OAuth.    

**Issues/challenges** 
Connect OAuth2.0 to integrate our project's login. 

**What I will work on** 
Housekeeping of project if anything that we missed or forgot, and create issues about them.

**Comment by OriginalJangster** [comment]
@OriginalJangster  Yuree Jang (n6m2b)

**What I did**
Implemented the drop down menu routing to accounts and inbox page (#53  ).
Implemented user Account page (#66 ), including functionality to switch between business owner and volunteer view, and rendering request list/ businesses owned vs. tasks/ privacy preference, respectively.

**Issues/challenges**
Resolving a lot of merge conflicts and having issues getting my repo to run after pulling changes.
Figuring out how to finish a task when it depends on other parts of the project being setup, and generally knowing what information I need and when/where to get it from to finish my part during the setup of most of the project's other parts (database, authentication, etc.).

**What I will work on**
Helping teammates wherever necessary (with deployment or implementation of any remaining pieces of the project).
Helping to do code reviews on pull requests.
